src/keymap.rs

```diff
@@ -562,8 +562,8 @@ impl<'b> InputState<'b> {
                     Cmd::CompleteBackward
                 }
             }
-            // Don't complete hints when the cursor is not at the end of a line
             E(K::Right, M::NONE) if wrt.has_hint() && wrt.is_cursor_at_end() => Cmd::CompleteHint,
+            E(K::Right, M::ALT) if wrt.has_hint() => Cmd::CompleteHint,
```

Comment:
I would prefer not to have this default binding in `rustyline`.
See `ConditionalEventHandler` to add your own custom binding.

Reply:
Yes, I have seen that API. I can remove that commit from the pull request, if that's your preference. Just thought that it gives a better out of the box experience for everybody.

Reply:
Using Alt+Right to get the hint completed also feels quite natural to me. With the rustyline default Alt+Right jumps to the end of the word. And that's exactly what happens when the hint is displayed right next to the cursor and Alt+Right is used.

Reply:
See https://github.com/kkawakam/rustyline/issues/406 where Ctrl-E feels quite natural to someone else...